client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -314,14 +347,22 @@ const percentCartModel = {
 		labelText: translate( 'Coupon Code & Discount' ),
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
-			couponCode: couponCodeField,
-			percentDiscount: (
-				<PercentField
-					labelText={ translate( 'Percent Cart Discount', { context: 'noun' } ) }
-					isRequired
-				/>
-			),
-			freeShipping: freeShippingField,
+			couponCode: {
+				component: couponCodeField,
+				validate: validateCouponCode,
+			},
+			percentDiscount: {
+				component: (
+					<PercentField
+						labelText={ translate( 'Percent Cart Discount', { context: 'noun' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.